### PR TITLE
RAB-934: migration add automation column in job_instance

### PR DESCRIPTION
**Pull request by MadeWilson**
**Description (for Contributor and Core Developer)**

add a new column to store automation configuration in job_instance table

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer

### CODE REVIEW

**Comment by grena** [comment]
So we should add a second indexed column `scheduled`, right?

File: upgrades/schema/Version_7_0_20220802151250_add_automation_column_in_job_instance.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+namespace Pim\Upgrade\Schema;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version_7_0_20220802151250_add_automation_column_in_job_instance extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql(<<<SQL
```
Comment (StevenVAIDIE) [inline upgrades/schema/Version_7_0_20220802151250_add_automation_column_in_job_instance.php:23]: The migration should be idempotent, we should check if column exist before adding column

**Comment by ghoussard** [comment]
> So we should add a second indexed column `scheduled`, right?

It is not under `automation` key like in the Spike ?

**Comment by grena** [comment]
Closed in favor of #17616

**Pull request closed without merging**